### DISCUSSION TITLE
Database backup documentation

### DISCUSSION THREAD

**Discussion by thigg** [nextcloud/all-in-one#2037] [Ideas] [answered]
### Is your feature request related to a problem? Please describe. 
I set my storage dir to a mounted smb share which is backed up. I do not want to backup my files again.
However as far as I can tell, this does not backup the database.

I couldn't find anything regarding that in the Documentation and I wonder if it is possible to only backup the database.

### Describe the solution you'd like
Run daily backup and upgrade but only backup the database.

### Describe alternatives you've considered
disabling backups completly

**Comment by szaimen** [comment, marked as answer]
Hi, since you seemingly don't want to use the built-in backup solution, you may have a look at this: https://github.com/nextcloud/all-in-one#how-to-enable-automatic-updates-without-creating-a-backup-beforehand

Key is to stop the containers correctly, then back up all the docker volumes and then start them again. Here are the possible options documented https://github.com/nextcloud/all-in-one#how-to-stopstartupdate-containers-or-trigger-the-daily-backup-from-a-script-externally

**Reply by thigg** [reply]
Yes, I saw this section. But that doesn't backup the database, right?

**Reply by thigg** [reply]
Or is the database backed up, if i backup the containers? If yes, how do I do that without backing up the data?

**Reply by szaimen** [reply]
Since the database is stored in a docker volume, it actually does a backup of the database. It is your responsibility in that case to do it correctly.
